manager/printermanager.go

```diff
@@ -225,7 +225,7 @@ func (pm *PrinterManager) applyDiff(diff *lib.PrinterDiff, ch chan<- lib.Printer
 			log.InfoPrinterf(diff.Printer.Name+" "+diff.Printer.GCPID, "Registered in the cloud")
 
 			if pm.gcp.CanShare() {
-				if err := pm.gcp.Share(diff.Printer.GCPID, pm.shareScope); err != nil {
+				if err := pm.gcp.Share(diff.Printer.GCPID, pm.shareScope, "USER", true); err != nil {
```

Comment:
I just re-read the documentation. You're right, we don't want emails going out, to keep with current behavior and because people get enough emails. LGTM.